src/librehatti/suspense/views.py

```diff
@@ -925,4 +926,97 @@ def summary_page(request):
         'other_charge':other_charges,'rate':transport['rate']})
     content = get_template('suspense/summary.html')
     html_content = content.render(temp)
-    return HttpResponse(html_content)
\ No newline at end of file
+    return HttpResponse(html_content)
+
+def transport_bill(request):
+    if request.method == 'POST':
+        #return HttpResponse('hello')
+        session = request.POST['session']
+        #return HttpResponse('hello')
+        voucher = request.POST['voucher']
+        transport_object = Transport.objects.filter(session_id = session,voucher_no = voucher).\
+        values('vehicle_id__vehicle_no','kilometer','rate','date_of_generation','date','total')
+        l = []
+        l1 = []
+        l3 = []
+        for value in transport_object:
+            vehicle = value['vehicle_id__vehicle_no']
+            rate = value['rate']
+            date_of_generation = value['date_of_generation']
+            k = str(value['kilometer'])[1:-1].encode("ascii",'ignore')
+            j = k.split(',')
+            for b in j:
+                c = b.replace("u'","")
+                c = c.replace("'","")
+                l1.append(c)
+            d  = str(value['date'])[1:-1]
+            e = d.split(',')
+            for a in e:
+                c = a.replace("u'","")
+                c = c.replace("'","")
+                l.append(c)
+            distance = 0
+            for temp_var in l1:
+                total = rate * int(temp_var)
```

Comment:
On 2 January 2015 at 21:35, jasvir99 <notifications@github.com> wrote:
>
> Don't use variables like a,b,c.

Oops,Sorry missed this point.
Should I cancel the pull request and resend it?


-- 
Har Amrit Pal Singh
http://haramritpanesar.wordpress.com/